scripts/setMockData.js

```diff
@@ -0,0 +1,58 @@
+const getProposals = require('./helpers/get-proposals')
```

Comment:
Could you please run the linters in the entire project? There are some warnings that can be solved.